docker.go

```diff
@@ -0,0 +1,385 @@
+package main
```

Comment:
Try running `golint`, that should help explain my code style. Also give `gofmt` a try as well! :)